desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -40,6 +40,10 @@ module.exports = class BidForm extends ErrorHandlingForm
     val = @$('.max-bid').val()
     @saleArtwork.cleanBidAmount val if val
 
+  removeRegistrationFormAndBidImmediately: =>
+    @$('.bid-and-register-form').hide()
```

Comment:
I put this in a separate method just to make the intent explicit. I can't find anywhere else in force's code that the submitImmediately option appears but i realize this might be needless boilerplate.